lib/thumbs/pull_request_worker.rb

```diff
@@ -920,6 +920,10 @@ def comment_code_approvals
       code_reviews
     end
 
+    def remove_build_dir
+      FileUtils.mv(@build_dir, "#{@build_dir}.#{DateTime.now.strftime("%s")}")
```

Comment:
I see build_dir as /tmp/thumbs/<unique_dir_name> , so this will technically move the directory to be a child of the build directory, so would cause problems for projects that recursively perform actions w/i their build directory. I think it is better to move the directory up to /tmp/thumbs/garbage or the like.